--- src/PySide-1.2.2/sources/shiboken/libshiboken/sbkenum.cpp.orig	2015-08-22 11:09:49.144841400 +0200
+++ src/PySide-1.2.2/sources/shiboken/libshiboken/sbkenum.cpp	2015-08-22 11:11:31.192678200 +0200
@@ -237,7 +237,7 @@
         Py_RETURN_FALSE;
 }
 
-static Py_hash_t enum_hash(PyObject* pyObj)
+static long enum_hash(PyObject* pyObj)
 {
     Py_hash_t val = reinterpret_cast<SbkEnumObject*>(pyObj)->ob_value;
     if (val == -1)
